docs/docs/configuration/record.md

```diff
@@ -42,3 +42,16 @@ The same options are available with events. Let's consider a scenario where you
 - With the `all` option all segments for the duration of the event would be saved for the event. This event would have 4 hours of footage.
 - With the `motion` option all segments for the duration of the event with motion would be saved. This means any segment where a car drove by in the street, person walked by, lighting changed, etc. would be saved. 
 - With the `active_objects` it would only keep segments where the object was active. In this case the only segments that would be saved would be the ones where the car was driving up, you going inside, you coming outside, and the car driving away. Essentially reducing the 4 hours to a minute or two of event footage. 
+
+A configration example of the above retain modes where `motion` is stored for 7 days and `active objects` are store for 14 would be as follows:
```

Comment:
I think this is a good change but could be reworded slightly to make things more clear.

"above retain modes where `motion` is stored for 7 days" -> "above retain modes where all `motion` segments are stored for 7 days"

"and `active objects` are store for 14" -> "and `active objects` events are stored for 14 days"

Reply:
Thanks looks good. I think it would be better to put the object specific retention separately so as to not confuse. Probably would fit under its own `## Object Specific Recording` header since it doesn't necessarily fit with the other topic

Reply:
What about `### Object Specific Retention` instead? And therefore be a part of the `## What do the different retain modes mean?` header, rather than it's own separate topic?

Reply:
Looks good to me